src/UnitTest/test_quote_prov.cpp

```diff
@@ -509,6 +526,17 @@ static void GetVerificationCollateralTest()
         &collateral);
     ASSERT_TRUE(SGX_QL_SUCCESS == result);
     VerifyCollateral(collateral);
+
+    collateral = nullptr;
+    result = sgx_ql_get_quote_verification_collateral_with_params(
+            TEST_FMSPC,
+            sizeof(TEST_FMSPC),
+            "processor",
+            custom_param,
+            custom_param_length,
+            &collateral);
+    ASSERT_TRUE(SGX_QL_SUCCESS == result);
+    VerifyCollateral(collateral);
```

Comment:
Additional verification will be needed. Would want to confirm that the collateral received has the same tcbevaluation number as what was requested.